packages/eas-cli/src/credentials/ios/appstore/AppStoreApi.ts

```diff
@@ -56,7 +56,7 @@ export default class AppStoreApi {
   public async ensureUserAuthenticatedAsync(options?: AuthenticateOptions): Promise<UserAuthCtx> {
     if (this.authCtx && !isUserAuthCtx(this.authCtx)) {
       // already authenticated, but with the wrong type
-      Log.log(`Only user authentication is supported. Reauthenticating as user...`);
+      Log.log(`Only user authentication is supported. Re-authenticating as user...`);
```

Comment:
```suggestion
      Log.log(`Only user authentication is supported. Reauthenticating as user...`);
```
This is the only suggested change I'd like to exclude.